pkg/event/module.go

```diff
@@ -154,28 +159,108 @@ func NewEventsDBModule(db DBEvents) EventsDBModule {
 
 func (cm *eventsDBModule) Store(_ context.Context, m map[string]string) error {
 
-	// prepare the query to insert the event in the DB
-	origin := "keycloak" // for the moment only events of Keycloak
-	realmName := m["realmId"]
-	agentUserID := ""   // no agents, only events from Keycloak
-	agentUsername := "" // no agents, only events from Keycloak
-	userID := m["userId"]
-	username := m["username"]
-	ctEventType := ""
-	kcEventType := m["type"]
-	kcOperationType := m["operationType"]
-	clientID := m["clienId"]
-	additionalInfo := "" // all the rest in a JSON?
-	/*if m["type"] == "LOGIN_ERROR" {
-		fmt.Println("The rest of event:")
-		fmt.Println(m)
-	}*/
-
-	_, err := cm.db.Exec(insertEvent, origin, realmName, agentUserID, agentUsername, userID, username, ctEventType, kcEventType, kcOperationType, clientID, additionalInfo)
+	// if ctEventType is not "", then record the events in MariaDB
+	if m["ct_event_type"] != "" {
+		origin := "keycloak" // for the moment only eventss of Keycloak
```

Comment:
typo: eventss

Reply:
for all the otheers, the comments is jsut before the line, not at the end. (Just for cosmetic ;) )